src/server/scripts/Commands/cs_ahbot.cpp

```diff
@@ -146,12 +146,28 @@ class ahbot_commandscript : public CommandScript
     static bool HandleAHBotRebuildCommand(ChatHandler* /*handler*/, const char* args)
     {
         char* arg = strtok((char*)args, " ");
+        char* ahMapIdStr = strtok(NULL, " ");
+        uint32 ahMapID = MAX_AUCTION_HOUSE_TYPE;
 
         bool all = false;
         if (arg && strcmp(arg, "all") == 0)
             all = true;
 
-        sAuctionBot->Rebuild(all);
+        if (ahMapIdStr)
+        {
+            // NOTE: Should use _stricmp() for case insenstive, stricmp() is deprecated
```

Comment:
You can just remove this comment, pretty much ALL posix functions with _ prefix are msvc specific, we ignore that warning for a reason.

Also - linux case: https://github.com/TrinityCore/TrinityCore/blob/6.x/src/common/Common.h#L79